website/assets/js/pages/articles/articles.page.js

```diff
@@ -59,6 +59,10 @@ parasails.registerPage('articles', {
           this.articleCategory = 'Reports';
           this.categoryDescription = '';
           break;
+        case 'case-studies':
+          this.articleCategory = 'Case studies';
+          this.categoryDescription = 'Read about how others are using Fleet and osquery.';
```

Comment:
@mike-j-thomas This is where the category name and description are set for the new article category page (`/case-studies`).